Flaky `TestBigQueryAvroConnectorTest.testQueryLabeling`

```sh
Error:  io.trino.plugin.bigquery.TestBigQueryAvroConnectorTest.testQueryLabeling  Time elapsed: 29.689 s  <<< FAILURE!
java.lang.AssertionError: 

Expected size: 1 but was: 0 in:
[]
	at io.trino.plugin.bigquery.BaseBigQueryConnectorTest.assertLabelForTable(BaseBigQueryConnectorTest.java:679)
	at io.trino.plugin.bigquery.BaseBigQueryConnectorTest.testQueryLabeling(BaseBigQueryConnectorTest.java:657)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

--- a/plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java
+++ b/plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java
@@ -679,10 +679,10 @@ public abstract class BaseBigQueryConnectorTest
                         SELECT * FROM UNNEST(labels) AS label WHERE label.key = 'trino_query' AND label.value = '%s'
                     )""".formatted(expectedLabel);
 
-        assertThat(bigQuerySqlExecutor.executeQuery(checkForLabelQuery).getValues())
+        assertEventually(() -> assertThat(bigQuerySqlExecutor.executeQuery(checkForLabelQuery).getValues())
                 .extracting(values -> values.get("query").getStringValue())
                 .singleElement()
-                .matches(statement -> statement.contains(expectedView));
+                .matches(statement -> statement.contains(expectedView)));
     }
 
     @Test

